tests/gtest/avif_fuzztest_helpers.h

```diff
@@ -40,6 +40,11 @@ AvifDecoderPtr CreateAvifDecoder(avifCodecChoice codec_choice, int max_threads,
                                  uint32_t image_dimension_limit,
                                  uint32_t image_count_limit,
                                  avifStrictFlags strict_flags);
+#if defined(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
+AvifDecoderPtr AddExperimentalOptionsToDecoder(
```

Comment:
maybe rename to AddExperimentalGainMapOptionsToDecoder or AddGainMapOptionsToDecoder ?